JPS_Ontology/ontology/ontopowsys/OntoPowSys.owl

```diff
@@ -78,7 +78,13 @@
         <rdfs:subPropertyOf rdf:resource="http://www.theworldavatar.com/ontology/meta_model/mereology/mereology.owl#isComposedOf"/>
     </owl:ObjectProperty>
     
+    <!-- http://www.theworldavatar.com/ontology/ontopowsys/OntoPowSys.owl#hasBusNode -->
 
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/ontology/ontopowsys/OntoPowSys.owl#hasBusNode">
+        <rdfs:subPropertyOf rdf:resource="http://www.theworldavatar.com/ontology/meta_model/fundamental_concepts/fundamental_concepts.owl#inter-objectRelation"/>
+        <rdfs:domain rdf:resource="http://www.w3.org/2002/07/owl#Thing"/>
```

Comment:
According to this, _anything_ can have a bus node. Any chance this can be made more specific?

Reply:
I have specified the domain of this newly created object property.